src/managers/input/InputManager.cpp

```diff
@@ -624,7 +624,7 @@ void CInputManager::processMouseDownNormal(wlr_pointer_button_event* e) {
     }
 
     switch (e->state) {
-        case WLR_BUTTON_PRESSED:
```

Comment:
?

Reply:
e->state is a wl_pointer_button_state, WLR_BUTTON_PRESSED is a wlr_button_state. 
it's comparing the wrong enums.
this stuff gives a warning in the editor so i fixed it

Reply:
ops